sw/device/tests/sensor_ctrl_alerts.c

```diff
@@ -49,36 +49,6 @@ static dif_alert_handler_t alert_handler;
 __attribute__((section(".non_volatile_scratch")))
 const volatile uint32_t events_vector = -1;
```

Comment:
do we want to declare the count somewhere else as well?

Reply:
I would prefer to leave it in each test... you can imagine tests that have multiple counters, for example.

Reply:
o good point.